app/react/Metadata/components/MetadataFormButtons.js

```diff
@@ -168,6 +170,7 @@ MetadataFormButtons.defaultProps = {
   formName: 'metadataForm',
   delete: () => {},
   copyFrom: () => {},
+  storeKey: undefined,
```

Comment:
I took a look and the thing is that the MetadataFormButtons is used also in context where there is no storeKey (like the EntityViewer). Formerly, there wasn't even a need to pass a storeKey, but know we need it to dispatch actions when there is a storeKey.
These actions are meant to be designed so that they only have the implied effect if there is a storeKey available. Regarding this, I found a bug that I'll fix.